fe/fe-core/src/main/java/com/starrocks/task/PublishVersionTask.java

```diff
@@ -159,14 +159,22 @@ public void updateReplicaVersions(List<TTabletVersionPair> tabletVersions) {
             LOG.warn("db not found dbid={}", dbId);
             return;
         }
+        List<Long> droppedTablets = new ArrayList<>();
+        for (int i = 0; i < tabletVersions.size(); i++) {
+            if (replicas.get(i) == null) {
+                droppedTablets.add(tabletVersions.get(i).tablet_id);
+            }
+        }
+        if (!droppedTablets.isEmpty()) {
```

Comment:
The WARNING log may misunderstand the user. I think the behavior is normal when some tables are dropped before publishing.

Reply:
changed